sql/dolphinscheduler_mysql.sql

```diff
@@ -471,7 +471,7 @@ CREATE TABLE `t_ds_task_definition` (
   `create_time` datetime NOT NULL COMMENT 'create time',
   `update_time` datetime DEFAULT NULL COMMENT 'update time',
   PRIMARY KEY (`id`,`code`),
-  UNIQUE KEY `task_unique` (`name`,`project_code`) USING BTREE
+  UNIQUE KEY `task_unique` (`code`,`project_code`) USING BTREE
```

Comment:
It seems the `code` is generated by a snowflake algorithm, we may don't need to add such a unique key.

Reply:
In the case of clock rollback, duplication may occur, so it is best to + unique constraint

Reply:
OK

Reply:
Thanks for your contribution. It can't be modified here. When the master module generates a DAG, it will use the name of the task as the key of the nodeMap (See MasterExecThread|buildFlowDag). Using code as the unique key will lead to the loss of the task. I'm refactoring this part. Please pay attention to json_split_two branch.

Reply:
As discussed in #5875, the design needs to be discussed. You can discuss how to make changes